libs/designsystem/src/lib/scss/base/_typography.scss

```diff
@@ -101,3 +101,20 @@ p,
 .kirby-text-bold {
   font-weight: font-weight('bold');
 }
+
+a {
+  $icon-scaling-factor: 1.5em;
+  color: inherit;
+  cursor: pointer;
+  text-decoration: underline;
+
+  &[target='_blank'] {
+    white-space: nowrap;
+    background-image: url('/assets/kirby/icons/svg/link.svg');
+    background-repeat: no-repeat;
+    background-position: right 50%;
+    background-size: $icon-scaling-factor;
+    //place icon to the right of text, with icons own width + the actual spacing
+    padding-right: calc(#{$icon-scaling-factor} + #{size('xxxs')});
+  }
+}
```

Comment:
👍

Reply:
A link is an interactive, focusable element. Have you considered what the `:focus` styles are? How does it look with the current implementation?

Reply:
I think it looks good with the current implementation. We can consider if it should be better aligned between all the components at a later time, but right now it matches what is on e.g. radio button.